### PR TITLE
[NG] Updating Home Page on layer5-ng 

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR updates Home page on layer5-ng website.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Banner-default/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import VintageBox from "../../reusecore/VintageBox";
 import ParticleComponent from "../../reusecore/Particle";
 
 import { FaPlay, FaDownload } from "react-icons/fa";
```
Comment (Nikhil-Ladha) [inline src/sections/Banner-default/index.js:9]: Please remove unused imports.

**Comment by Nikhil-Ladha** [comment]
Are you planning to update the complete page in iterations or in this PR itself?

**Comment by Nikhil-Ladha** [comment]
The whitespace here can be reduced a bit.
Screenshot from 2020-12-03 13-10-39

**Comment by Nikhil-Ladha** [comment]
Also, isn't there any SVG for the `Life cycle management` stage?

**Comment by Jashpatel1** [comment]
> Also, isn't there any SVG for the `Life cycle management` stage?

Only the first two sections(banner and projects) have been updated. Rest design is to be approved yet.

File: src/sections/Projects-home/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <Container>
+                <Row>
+                    <div className="project-title">
+                        <SectionTitle
+                            className="section-title"
+                            leftAlign={true}
+                            UniWidth="75%"
+                        >
+                            <h4>PROJECTS</h4>
+
```
Comment (Nikhil-Ladha) [inline src/sections/Projects-home/index.js:29]: This _empty_ line can be removed

**Comment by Nikhil-Ladha** [comment]
> Only the first two sections(banner and projects) have been updated. Rest design is to be approved yet.

Yeah, I just noticed that!

**Comment by Nikhil-Ladha** [comment]
Can we add a hover effect for the projects card?

**Comment by Tanuj22** [comment]
+1 on the hover effect. Rest looks good

**Review by Tanuj22 — APPROVED** [review]
LGTM

**Pull request merged**